src/overview/index.ts

```diff
@@ -81,8 +81,17 @@ async function initializeOverviewView(context: vscode.ExtensionContext, webviewP
       toggleOverviewVisibilityOperation(context, e.visibility);
     } else if (e.command === "installExtension") {
       await vscode.commands.executeCommand("java.helper.installExtension", e.extName, e.displayName);
-    } else if (e.command === "java.gettingStarted") {
-      await vscode.commands.executeCommand("java.gettingStarted");
+    } else if (e.command) {
```

Comment:
Will you also refactor the way we call `installExtension`?

Reply:
addressed.